adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java

```diff
@@ -82,8 +82,9 @@
      * Check case-insensitive lookup
      */
     private static final String VALID_AUTHORITY = "https://login.windows.net/test.onmicrosoft.com";
-    private static final int ACTIVITY_TIME_OUT = 1000;
+    private static final int ACTIVITY_TIME_OUT = 2000;
```

Comment:
But this will cause all other tests to be running slower. Can you just do sync call? then there is no need to have the countdownlatch.